test_projects/cnode/c_src/example/example.spec.exs

```diff
@@ -49,3 +49,7 @@ type nested_struct :: %Nested.Struct{
 }
 
 spec test_nested_struct(in_struct :: nested_struct) :: {:ok :: label, out_struct :: nested_struct}
+
+type my_enum :: :option_one | :option_two | :option_three | :option_four | :option_five
+
+spec test_my_enum(in_enum :: my_enum) :: {:ok :: label, out_enum :: my_enum}
```

Comment:
Is it possible to write spec like

```elixir
spec test_my_enum(in_enum :: my_enum) ":: {:error :: label, :not_found}
```
I mean, can we write atoms directly in specs as stated in #61?

If no is it easy to introduce this? We can also make an issue for this.

Reply:
I would create separate issue for this conception, to have a place to discuss some doubts, like how to access these inlined atoms/enums from code in C/C++

Reply:
Sure, so please create a new issue for this and link it here ;)

Reply:
Ping